components/user-mgt/org.wso2.carbon.user.mgt.workflow/src/main/java/org/wso2/carbon/user/mgt/workflow/userstore/UserStoreActionListener.java

```diff
@@ -46,15 +46,15 @@ public int getExecutionOrderId() {
     @Override
     public boolean doPreAddUser(String userName, Object credential, String[] roleList, Map<String, String> claims,
                                 String profile, UserStoreManager userStoreManager) throws UserStoreException {
+
         if (!isEnable()) {
             return true;
         }
-
         try {
+            AddUserWFRequestHandler addUserWFRequestHandler = new AddUserWFRequestHandler();
```

Comment:
No, This was fixed adding some more and then revert back to original code. There were no any issues in previous implementation.
But In this code we need to concern more stuff than this single line. Because we don't need to create object even. It is already registered as OSGi and we need to get it. We will fix those thing as improvement.